vtr_flow/benchmarks/tests/test_eblif.eblif

```diff
@@ -5,12 +5,16 @@
 .names a b a_and_b
 11 1
 .cname lut_a_and_b
-.param test_names_param "test_names_param_value"
+.param test_names_param_str "test_names_param_str_value"
+.param test_names_param_bin 00110101
+.param test_names_param_num 2.0
 .attr test_names_attrib "test_names_param_attrib"
 
 .latch a_and_b dff_q re clk 0
 .cname my_dff
-.param test_latch_param "test_latch_param_value"
+.param test_latch_param "test_latch_param_str_value"
+.param test_latch_param_bin 00110101
+.param test_latch_param_num 2.0
```

Comment:
You need negative tests here!

Reply:
Also I recommend reviewing the failure output to make sure it is possible to understand the problem from the error message.  Concrete things I'd look for:
 - Is the location of the error made clear (e.g. file and line number)
 - Is there an explanation of why the input was wrong?
  - For example, missing quotes
  - A bare integer with no period

Reply:
I believe that VPR at this point doesn't support test cases that are meant to fail (however https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/scripts/python_libs/vtr/util.py#L73-L75 allows that but this parameter is not used anywhere in the test framework).
Anyway, I tested the error output and modified it to provide more useful information.